specs/file_share_list_view.adoc

```diff
@@ -0,0 +1,143 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of file shares present in Gluster cluster in Tendrl
+
+Display list of file shares associated with a Gluster cluster present in Tendrl. The list should have functionality for 
+
+* filtering by name, cluster,status and type.
+* sorting by name and utilisation.
+
+File share list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of file share associated with a gluster cluster along with their details.
+
+The details shown for each file share in UI is -
+
+* Name
+* Status
+* 2-way/3-way distributed
+* Cluster Name
+* CPU Utilisation
+* Bricks
+* Active alerts
+* Last Rebalance date
+
+API support present for -
+
+* Name
+* Status
+* Cluster Name
+* CPU Utilisation
+* Bricks
+
+== Use Cases
+
+Users can see the list of file shares associated with a gluster cluster present in Tendrl. They can also 
+
+* filter by name, cluster, status and type.
+* sort by name and utilisation.
+
+As per API support, users can 
+
+* filter by name, cluster and status.
+* sort by name and utilisation.
+
+== Proposed change
+
+* Create a view for listing file share associated with a gluster cluster.
+* For utilisation, use donut chart component.
```

Comment:
for donut chart we need specify the height and width also the width of donut. So it can be consistent across the views.

Example:
DonutChartConfig.size.height = 90; 
DonutChartConfig.size.width = 80;
DonutChartConfig.donut.width = 10; //This is width of the donut ring we will be using.